app.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  get,
+  param,
+  FindRoute,
+  ParseParams,
+  InvokeMethod,
+  Send,
+  Reject,
+  RestBindings,
+  SequenceHandler,
+  ParsedRequest,
+  ServerResponse,
+  RestComponent,
+  RestServer,
+} from '@loopback/rest';
+import {
+  LogComponent,
+  log,
+  LogBindings,
+  LogFn,
+  LogLevel,
+  LogLevelMixin,
+} from './src';
+import {inject} from '@loopback/context';
+import {Application} from '@loopback/core';
+
+const restSequenceActions = RestBindings.SequenceActions;
+
+class MyController {
```

Comment:
Controllers should be defined in `src/controllers/*.controller.ts`.

I am little bit confused about the purpose of this sample controller and application. Isn't this repository supposed to provide an extension, not an application?